cmd/atlantis.go

```
@@ -10,6 +11,9 @@ var atlantisCmd = &cobra.Command{
 	Short:              "Execute 'atlantis' commands",
 	Long:               `This command executes Atlantis integration commands`,
 	FParseErrWhitelist: struct{ UnknownFlags bool }{UnknownFlags: false},
+	Run: func(cmd *cobra.Command, args []string) {
+		exec.ExecuteAtlantis(cmd, args)
```

golangci-lint (1.62.2)

Error return value of `exec.ExecuteAtlantis` is not checked

(errcheck)